File: PassingCars/ruby/TestPassingCars.rb
require_relative 'PassingCars'
require 'minitest/autorun'

class Test < MiniTest::Test

  def setup
    @passing_cars = PassingCars.new
  end

  def test_solution
    assert_equal 5, @passing_cars.solution([0,1,0,1,1])
  end

end

File: PassingCars/php/TestPassingCars.php
<?php

require 'PassingCars.php';

class Test extends PHPUnit_Framework_TestCase {

	protected $passing_cars;

	protected function setup() {
		$this->passing_cars = new \Codility\PassingCars;
	}

	public function testSolution() {
		$this->assertEquals(5, $this->passing_cars->solution([0,1,0,1,1]));
	}

}